server/routes/songs.js

```diff
@@ -11,6 +11,7 @@ let globallySearchedSongs = [
     "albumName": "Attention",
     "albumImgSrc": "https://i.scdn.co/image/ab67616d0000b27324b893fb9e7953ebc9517c6a",
     "songPreview": "https://p.scdn.co/mp3-preview/0bc92811cc92e03805da4a1a5e8d70a0dee73107?cid=07f17a9780d34fc5aac6cb73f0ca6ce8",
+    "externalUrl": "https://open.spotify.com/track/11xC6P3iKYpFThT6Ce1KdG",
```

Comment:
something like this, similar to sample.js?


```
name: "Attention",
artists: [{ name: "Doja Cat" }],
album: {
      images: [
        {
          url: "https://i.scdn.co/image/ab67616d0000b27324b893fb9e7953ebc9517c6a",
        },
      ],
    },
preview_url:
      "https://p.scdn.co/mp3-preview/0bc92811cc92e03805da4a1a5e8d70a0dee73107?cid=07f17a9780d34fc5aac6cb73f0ca6ce8",
```

Reply:
I've remodelled the API to match spotify's :)